src/alfasim_sdk/_internal/alfacase/alfacase_to_case.py

```diff
@@ -1731,6 +1731,26 @@ def load_wall_description(
     ]
 
 
+def load_leak_equipment_description(
+    document: DescriptionDocument,
+) -> List[case_description.LeakEquipmentDescription]:
+    alfacase_to_case_description = get_case_description_attribute_loader_dict(
+        case_description.LeakEquipmentDescription
+    )
+
+    def generate_leak_equipment_description(document: DescriptionDocument):
+        case_values = to_case_values(document, alfacase_to_case_description)
+        item_description = case_description.LeakEquipmentDescription(**case_values)
+        return update_multi_input_flags(document, item_description)
+
+    return {
+        key.data: generate_leak_equipment_description(
```

Comment:
Food for thought.
I ask myself why `DescriptionDocument` do not have an `items` method.

```python

@attr.s
class DescriptionDocument:
    ...
    def items(self):
        for key, value in self.content.items():
            yield key, type(self)(value, self.file_path)
```

Probably will be expected to have other helper methods.